tests/framework/extensions/provisioninginput/config.go

```diff
@@ -193,28 +193,29 @@ type NodePools struct {
 }
 
 type Config struct {
-	NodePools              []NodePools                              `json:"nodePools,omitempty" yaml:"nodePools,omitempty"`
```

Comment:
did this rebase correctly? Looks like all the 'omitempty' fields were removed

Reply:
if so, could you add the omitempty field to CloudProvider?

Reply:
Good catch, fixing now

Reply:
Fixed and added the field to CloudProvider.